src/desktop/components/react/stitch_components/StitchWrapper.tsx

```diff
@@ -3,14 +3,18 @@ import { data as sd } from "sharify"
 import { SystemContextProvider } from "v2/Artsy"
 import { MediaContextProvider } from "v2/Utils/Responsive"
 import { Theme } from "@artsy/palette"
+import { FocusVisible } from "v2/Components/FocusVisible"
 
 const mediator = require("desktop/lib/mediator.coffee")
 
 export const StitchWrapper = props => {
   return (
     <Theme>
       <SystemContextProvider user={sd.CURRENT_USER} mediator={mediator}>
-        <MediaContextProvider>{props.children}</MediaContextProvider>
+        <MediaContextProvider>
+          <FocusVisible />
```

Comment:
@damassi Sort of weird we're not including GlobalStyles here as well?

Reply:
Yah, this is interesting, but nothing has come up in the past. But the issue was probably not noticed? Feel free to add if it feels right looking at the nav bar on old pages.